Index: Java101/TaksimetreProgrami/src/TaksimetreProgrami.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Java101/TaksimetreProgrami/src/TaksimetreProgrami.java b/Java101/TaksimetreProgrami/src/TaksimetreProgrami.java
new file mode 100644
--- /dev/null	(date 1683490359678)
+++ b/Java101/TaksimetreProgrami/src/TaksimetreProgrami.java	(date 1683490359678)
@@ -0,0 +1,21 @@
+package TaksimetreProgrami;
+
+import java.util.Scanner;
+public class TaksimetreProgrami {
+    public static void main(String[] args) {
+        int km;
+        double perKm = 2.20, total, startPrice = 10;
+
+        Scanner input = new Scanner(System.in);
+        System.out.print("Mesafeyi KM cinsinden giriniz : ");
+        km = input.nextInt();
+
+        total = (km * perKm);
+        total += startPrice;
+
+        total = ( total < 20) ? 20 : total;
+        System.out.println("Toplam Tutar : " + total + " TL");
+
+    }
+}
+
