Index: Java101/TaksimetreProgrami/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Java101/TaksimetreProgrami/.gitignore b/Java101/TaksimetreProgrami/.gitignore
deleted file mode 100644
--- a/Java101/TaksimetreProgrami/.gitignore	(revision 5e0b9570946a2726a687f93b47aceeee90958215)
+++ /dev/null	(revision 5e0b9570946a2726a687f93b47aceeee90958215)
@@ -1,29 +0,0 @@
-### IntelliJ IDEA ###
-out/
-!**/src/main/**/out/
-!**/src/test/**/out/
-
-### Eclipse ###
-.apt_generated
-.classpath
-.factorypath
-.project
-.settings
-.springBeans
-.sts4-cache
-bin/
-!**/src/main/**/bin/
-!**/src/test/**/bin/
-
-### NetBeans ###
-/nbproject/private/
-/nbbuild/
-/dist/
-/nbdist/
-/.nb-gradle/
-
-### VS Code ###
-.vscode/
-
-### Mac OS ###
-.DS_Store
\ No newline at end of file
Index: Java101/TaksimetreProgrami/TaksimetreProgrami.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Java101/TaksimetreProgrami/TaksimetreProgrami.iml b/Java101/TaksimetreProgrami/TaksimetreProgrami.iml
deleted file mode 100644
--- a/Java101/TaksimetreProgrami/TaksimetreProgrami.iml	(revision 5e0b9570946a2726a687f93b47aceeee90958215)
+++ /dev/null	(revision 5e0b9570946a2726a687f93b47aceeee90958215)
@@ -1,11 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<module type="JAVA_MODULE" version="4">
-  <component name="NewModuleRootManager" inherit-compiler-output="true">
-    <exclude-output />
-    <content url="file://$MODULE_DIR$">
-      <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
-    </content>
-    <orderEntry type="inheritedJdk" />
-    <orderEntry type="sourceFolder" forTests="false" />
-  </component>
-</module>
\ No newline at end of file
